src/robot/libraries/String.py

```diff
@@ -121,6 +123,45 @@ def decode_bytes_to_string(self, bytes, encoding, errors='strict'):
             raise TypeError('Can not decode strings on Python 3.')
         return bytes.decode(encoding, errors)
 
+    def format_string(self, template, *positional, **named):
+        """Formats a ``string`` using the given ``positional`` and ``named``
```

Comment:
The parameter name is "template", but it is referenced as "string" in the docstring. I'm not sure how will this turn up in the generated documentation (haven't ever done it myself), but won't it lead to a difference - "template" in the keyword signature (the left column), and "string" in the documentation body (the right column)?

If so, that can be a bit confusing; it'd be better if the to names are the same, e.g.
"Formats a \`\`template\`\` (a string, or a file's content) using the ..."

Reply:
Yeah, should use `string` in the doc. Don't think it's necessarily to explain already here hat the template can be a string or a path to a file, though.

Reply:
Sorry @pekkaklarck did you actually meant to say "should use `template` in the doc"?

Reply:
Yes, `template` is better as an argument name and should then obviously be used in the documentation as well.